src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/MultiSigWidget/utils.ts

```diff
@@ -17,29 +18,27 @@ export const hasWeekPassed = (createdAt: string) => {
 };
 
 export const getNotSignedUsers = ({
-  requiredRoles,
   signatures,
   eligibleSignees,
 }: {
   signatures: MultiSigUserSignature[];
-  eligibleSignees: User[];
-  requiredRoles: ColonyRole[];
+  eligibleSignees: EligibleSignee[];
 }): MultiSigSignee[] => {
   const notSignedUsers: MultiSigSignee[] = eligibleSignees
     .filter((eligibleSignee) => {
       return !signatures.find(
-        (signature) => signature.userAddress === eligibleSignee?.walletAddress,
+        (signature) => signature.userAddress === eligibleSignee.userAddress,
       );
     })
```

Comment:
Would it make the code easier to follow if we create an array with the signature user addresses 
```
signatureUserAddresses = signatures.map(signature => signature.userAddress)
```

and then filter the eligibleSignees

```
 eligibleSignees
    .filter((eligibleSignee) => !signatureUserAddresses.includes(eligibleSignee.userAddress))
    .map(...)
```
?